shared/core/types/proto/relay.proto

```diff
@@ -18,12 +18,15 @@ message RelayPayload {
 }
 
 message RelayMeta {
+    reserved 5;
+    reserved "token";
     int64 block_height = 1;
     string servicer_public_key = 2;
     Identifiable relay_chain = 3;
     Identifiable geo_zone = 4;
-    AAT token = 5;
     string signature = 6;
+    // TODO: should we use public key instead?
+    string application_address = 7;
```

Comment:
Yes

Reply:
On a second review, we may need to keep this as `application_address`, as it is needed by `GetSession`